### PR TITLE
Add some issue templates

### PR DESCRIPTION
**Pull request by jonmeow**
ref: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

example of how this is supposed to look: https://github.com/flutter/flutter/issues/new/choose

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/01_explorer_bug.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      label: >
+        Description of the bug:
+  - type: textarea
+    id: repro
+    attributes:
+      label: >
+        What's the simplest, easiest way to reproduce this bug?
+      description: >
+        Please provide example code and errors; a shortlink to the execution on
+        carbon.compiler-explorer.com can also help.
```
Comment (zygoloid) [inline .github/ISSUE_TEMPLATE/01_explorer_bug.yaml]: I wonder if the structure of "What did you do?", "What did you expect to happen?" "What actually happened?" would be more likely to give us good information here. I could imagine someone filling out this form without ever saying why they think the observed behavior is a bug.

**Review by zygoloid — COMMENTED** [review]
Seems reasonable. I assume we'll still be able to file freeform bugs that don't use these templates?

**Comment by zygoloid** [comment]
> Seems reasonable. I assume we'll still be able to file freeform bugs that don't use these templates?

Answering my own question: yes, that's what the "allow blank issue" setting does.

File: .github/ISSUE_TEMPLATE/01_explorer_bug.yaml (hunk 1 above)
Comment (jonmeow) [reply .github/ISSUE_TEMPLATE/01_explorer_bug.yaml]: Done

**Review by zygoloid — APPROVED** [review]

**Pull request merged**